pkg/catalog/catalog.go

```diff
@@ -9,55 +9,69 @@ import (
 )
 
 // List lists all the available component types
-func List(client *occlient.Client) ([]string, error) {
+func List(client *occlient.Client) (map[string][]string, error) {
 	var catalogList []string
 	imageStreams, err := getDefaultBuilderImages(client)
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to get image streams")
 	}
 	catalogList = append(catalogList, imageStreams...)
-
+	finalList := make(map[string][]string)
+	finalList["imagestream"] = catalogList
 	// TODO: uncomment when component create supports template creation
-	//clusterServiceClasses, err := client.GetClusterServiceClassExternalNames()
-	//if err != nil {
-	//	return nil, errors.Wrap(err, "unable to get cluster service classes")
-	//}
-	//catalogList = append(catalogList, clusterServiceClasses...)
+	clusterServiceClasses, err := client.GetClusterServiceClassExternalNames()
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get cluster service classes")
+	}
+	var clusterSvcClass []string
+	for _, csClass := range clusterServiceClasses {
+		if !strings.Contains(csClass, "apb") {
```

Comment:
why is `apb` excluded?

Reply:
we can not deploy apb directly because there are some parameters which needs to be filled, whereas templates have default values for parameters.
cc @containscafeine